options/locale/locale_ru-RU.ini

```diff
@@ -1774,7 +1774,8 @@ auths.tip.google_plus=Получите учетные данные клиент
 auths.tip.openid_connect=Используйте OpenID Connect Discovery URL (<server>/.well-known/openid-configuration) для автоматической настройки входа OAuth
 auths.tip.twitter=Перейдите на https://dev.twitter.com/apps, создайте приложение и убедитесь, что включена опция «Разрешить это приложение для входа в систему с помощью Twitter»
 auths.tip.discord=Добавьте новое приложение на https://discordapp.com/developers/applications/me
-auths.tip.gitea=Зарегистрировать новое приложение OAuth2. Руководство можно найти на https://docs.gitea.io/ru-us/oauth2-provider/
+auths.tip.gitea=Зарегистрируйте новое приложение OAuth2. Руководство можно найти на https://docs.gitea.io/ru-us/oauth2-provider/
```

Comment:
Please revert changes in locales different from `en-US`. These locales are updated via [Crowdin](https://crowdin.com/project/gitea).